_posts/2022-08-08-WX_Blog1.md

```diff
@@ -1,59 +1,60 @@
+    Particle 1 will interact with particles 2-10, making it 9 interactions;
+    Particle 2 will interact with particles 3-10, making it 8 interactions;
+    so on and so forth, and the total number of interactions can be calculated as: 9 + 8 + 7 + ... + 1 = 45 interactions.
+
+For 100 particles, the equation can be similarly written as 99 + 98 + 97 + ... + 2 + 1 = 4950 interactions. Hei, Gauss solved it 300 years ago! Maybe we should be born earlier than him? ---quote by Gil Strand.
+For 1000 particles, similar extrapolation can be made as: 999 + 998 + 997 + ... + 2 + 1 = 499500 interactions.
+Hence a more scientificly appealing way to address the general format of pairwise particle interactions for n particles is:
+$$ 	N\left(n \right) = \frac{n * \left(n - 1\right)}{2} $$
+
+Do we seriously need to take all the universe of particle interactions into account? From the Lennard Jones approximation, one can easily figure out that is probably unecessary. Two particles, at free motion, can be extremely repulsive when they are close to each other. Imagine a unit density of 1 particle / unit volume, in any given sphere with a radius r, the volume of the sphere will be
+$$ 	V\left(r \right) = \frac{4 * \pi * r^3}{3} $$
+And the number of particles within the volume is:
+$$ 	N\left(r \right) = \rho * \frac{4 * pi * r^3}{3} $$
+
+For a radius of 1, N of particles is: 4
+For a radius of 2, N of particles is: 33
+For a radius of 5, N of particles is: 523
+so on and so forth.
+
+Now with all of the above thoughts, one should be able to come up with a hypotheis:
+it is unecessary to account for all the particle interactions, given their repulsion and density. It will be such an enormous amount of calculation that even the most powerful computer is dwarfed.
+The benefits of truncating unecessary interactions are: Reduce the cost of computation. Save time of effort.
+But on the other hand, the associated risks are: The method is less accurate. It might affect the final result. We need to be unscientific to move forward scientifically.
+
+# Re-walk the path of Bohr
+Bohr would've wanted to come up with a electron probability model, instead of the quantized orbital model, if he's offered further experimental data.
+I would've asked the computer to just keep calculating, especially after truncating a whole bunch of interactions. But it's still too much to compute.
+
+Yet another approximation is here for science. Intuitively, we can assume that if all the particles are randomly distributed (of course not in reality), after a large enough space is given, whatever particle movement inside the space should not affect the adjacent space (and certainly not the case, especially at the interface). I have to admit this part I'm not 100% satisfied. I wish to imagine these spaces / boxes like unit cell of crystal lattice, but with no fixed covalent bonding it's hard to imagine how uniform the spaces can be.
```

Comment:
Maybe mentioning periodic boundary and what it is will be helpful for the readers in this section since you're talking about it below.